src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/steps/HearingApiCallSteps.java

```diff
@@ -1,16 +1,15 @@
 package uk.gov.hmcts.futurehearings.hmi.unit.testing.steps;
 
+import static io.restassured.RestAssured.expect;
+
 import io.restassured.response.Response;
-import net.thucydides.core.annotations.Step;
 
 import java.util.Map;
 
-import static net.serenitybdd.rest.SerenityRest.expect;
 
 public class HearingApiCallSteps {
```

Comment:
I dont think we need this class now, pull this method up and remove this class if not required.